cityofaustin/atd-moped
# Refine Projects page advanced filters for date fields

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/16639

## Testing
https://deploy-preview-1318--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

1. Test with the following date fields : Construction start, Completion date, Modified
2. Do you see the new labels, do they make sense?
3. See and test the new is blank option.
4. Test using the "on" operator, previously that wasnt working correctly bc we were searching timestampz with a date string. Now it should return expected dates



---
#### Ship list
